api/candlepin-api-spec.yaml

```diff
@@ -2849,6 +2849,305 @@ paths:
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        403:
+          description: Users cannot access either of the consumer, owner, or pools
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Users cannot access either of the consumer, owner, or pools
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        404:
+          description: Specified consumer(unit) or owner is not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Specified consumer(unit) or owner is not found
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+  /pools/{pool_id}:
+    get:
+      description: Retrieves a single pool
+      tags:
+        - Pools
+      operationId: getPool
+      parameters:
+        - in: path
+          name: pool_id
+          description: Pool ID
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: consumer
+          description: Consumer UUID
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: activeon
+          description: Uses ISO 8601 format
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Retrives a single pool by its ID
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PoolDTO'
+        403:
+          description: Users cannot access consumer(unit)
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Users cannot access consumer(unit)
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        404:
+          description: Subscription pool or consumer(unit) not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Subscription pool or consumer(unit) not found
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+    delete:
+      description: Removes a pool
+      tags:
+        - Pools
+      operationId: deletePool
+      parameters:
+        - in: path
+          name: pool_id
+          description: Pool ID
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Retrives a single pool by its ID
```

Comment:
Wrong description and response code should be 204.

Reply:
Corrected.